plugin/src/main/resources/io/jenkins/plugins/coverage/metrics/restapi/_api.jelly

```diff
@@ -0,0 +1,17 @@
+<?jelly escape-by-default='true'?>
```

Comment:
I'm not sure if this is the right package for that file? I can't see the content. Do you see that text?

Reply:
Done. Has been in the wrong directory indeed.